spark-indexer/src/main/java/org/uniprot/store/spark/indexer/uniprot/converter/UniProtEntryCommentsConverter.java

```diff
@@ -58,30 +58,25 @@ class UniProtEntryCommentsConverter implements Serializable {
     void convertCommentToDocument(List<Comment> comments, UniProtDocument document) {
         for (Comment comment : comments) {
             FFLineBuilder<Comment> fbuilder = CCLineBuilderFactory.create(comment);
-            // TODO field is too generic, we can rename it to commentField
-            String field = getCommentField(comment);
+            String commentField = getCommentField(comment);
             String evField = getCommentEvField(comment);
-            // TODO value is too vague, commentValues will give more hint about the type values
-            // why are we using Collection type not List or Set
-            Collection<String> value =
-                    document.commentMap.computeIfAbsent(field, k -> new ArrayList<>());
+            Collection<String> commentValues =
+                    document.commentMap.computeIfAbsent(commentField, k -> new ArrayList<>());
 
             String commentVal = fbuilder.buildString(comment);
-            value.add(commentVal);
+            commentValues.add(commentVal);
             document.content.add(commentVal);
-            // TODO the name of the field evValue should be plural since this is a collection
-            Collection<String> evValue =
+            Collection<String> evValues =
                     document.commentEvMap.computeIfAbsent(evField, k -> new HashSet<>());
             Set<String> evidences = fetchEvidences(comment, document.reviewed);
-            evValue.addAll(evidences);
+            evValues.addAll(evidences);
 
             if (hasExperimentalEvidence(evidences)) {
-                String experimentalField = field + EXPERIMENTAL;
-                //  TODO the name should be plural
-                Collection<String> experimentalComment =
+                String experimentalField = commentField + EXPERIMENTAL;
+                Collection<String> experimentalComments =
                         document.commentMap.computeIfAbsent(
                                 experimentalField, k -> new ArrayList<>());
-                experimentalComment.add(commentVal);
+                experimentalComments.add(commentVal);
             }
 
```

Comment:
Could we replace this code block with  addExperimentalEvidence call?

Reply:
The method created uses Set, and this part uses a List. this is the reason I did not add for this